speed-dreams/source-2.2.3/data/data/menu/DeveloperMenu.xml

```diff
@@ -147,6 +147,29 @@
             <attstr name="pushed image" val="data/img/button-left-pushed.png"/>
         </section>
 
+
+        <!-- Default button-->
+        <section name="DefaultButton">
+	        <attstr name="type" val="text button"/>
+	        <attstr name="show box" val="no"/>
+	        <attstr name="text" val="Default"/>
```

Comment:
I'd suggest "Defaults" or "Load Defaults" or something similar

Reply:
changed it to "Set defaults"